pkgs/applications/networking/feedreaders/newsflash/default.nix

```diff
@@ -0,0 +1,68 @@
+{ lib, rustPlatform, fetchFromGitLab
+, gdk-pixbuf, glib, meson, ninja, pkg-config, wrapGAppsHook
+, gsettings-desktop-schemas, gtk3, libhandy, librsvg, openssl, sqlite
+, webkitgtk
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "newsflash";
+  version = "1.0.1";
+
+  src = fetchFromGitLab {
+    owner = "news-flash";
+    repo = "news_flash_gtk";
+    rev = version;
+    sha256 = "1y2jj3z08m29s6ggl8q270mqnvdwibs0f2kxybqhi8mya5pyw902";
+  };
+
+  cargoPatches = [
+    ./cargo.lock.patch
+  ];
```

Comment:
Could you open an issue upstream asking if they could add a lock file and add a comment here with a link to the issue?

Reply:
:+1: Will do